core/store/src/shard_uid_mapping.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{
+    flat::POISONED_LOCK_ERR,
+    io::{Error, Result},
+};
+use near_primitives::shard_layout::ShardUId;
+use std::{
```

Comment:
super nit: could you please set `"rust-analyzer.imports.granularity.group": "module"` for rust-analyzer?